Source/ACE/Network/Handlers/CharacterHandler.cs

```diff
@@ -140,7 +141,9 @@ public static void CharacterRestore(ClientPacketFragment fragment, Session sessi
             }
 
             CharacterCreateSetDefaultCharacterOptions(character);
+            CharacterCreateSetDefaultCharacterPositions(character);

```

Comment:
Set's the initial values for the character positions, when the character is first created.